ui/ui-config/pages/model/[modelId]/record/new.tsx

```diff
@@ -1,122 +0,0 @@
-import Head from "next/head";
```

Comment:
👍 the only way to add a record manually is when in ui-config, and using the modal now